.travis.yml

```diff
@@ -99,13 +105,23 @@ after_success:
       graphql-schema-linter ./schema.graphql
     fi
   # Generate the APIGen Docs
+  # Generate and Deploy the mkdocs for GithubPages
   - |
     if [ $APIGEN_DOCS == true ]; then
       cd $TRAVIS_BUILD_DIR
       vendor/bin/apigen generate --source="src" --destination $TRAVIS_BUILD_DIR/apigen
     fi
 
 deploy:
+  - provider: pages
+    github_token: $WPGRAPHQL_GITHUB_TOKEN
+    local_dir: $TRAVIS_BUILD_DIR/site
+    skip_cleanup: true
+    on:
+      # Probably switch this to master?
+      all_branches: develop
+      # Only deploy when APIGEN_DOCS are generated
+      condition: $APIGEN_DOCS == true
```

Comment:
@jasonbahl, all of this is duplicated again below, pretty sure this is what's breaking the travis build.